test/local/routes/sms.js

```diff
@@ -121,12 +122,17 @@ describe('/sms with the signinCodes feature included in the payload', () => {
         assert.equal(args[0].event, 'sms.installFirefox.sent')
         assert.equal(args[0].flow_id, request.payload.metricsContext.flowId)
       })
+
+      it('returns a formattedPhoneNumber', () => {
+        assert.equal(response.formattedPhoneNumber, '888-508-3401')
```

Comment:
The country code is not returned in the response, but we add that on the front end.